src/org/isf/medicals/service/MedicalsIoOperationRepository.java

```diff
@@ -11,48 +11,48 @@
 
 @Repository
 public interface MedicalsIoOperationRepository extends JpaRepository<Medical, Integer> {
-	@Query(value = "SELECT * FROM MEDICALDSR where MDSR_DESC like :description order BY MDSR_DESC", nativeQuery= true)
+	@Query(value = "SELECT m FROM Medical m where m.description like :description order BY m.description")
     List<Medical> findAllWhereDescriptionOrderByDescription(@Param("description") String description);
-	@Query(value = "SELECT * FROM MEDICALDSR JOIN MEDICALDSRTYPE ON MDSR_MDSRT_ID_A = MDSRT_ID_A order BY MDSR_DESC", nativeQuery= true)
+	@Query(value = "SELECT m FROM Medical m JOIN m.type mt order BY m.description")
     List<Medical> findAllByOrderByDescription();
-    @Query(value = "SELECT * FROM MEDICALDSR JOIN MEDICALDSRTYPE ON MDSR_MDSRT_ID_A = MDSRT_ID_A where MDSRT_DESC like :type order BY MDSR_DESC", nativeQuery= true)
+    @Query(value = "SELECT m FROM Medical m JOIN m.type mt where mt.description like :type order BY m.description")
```

Comment:
This will also work without join sth like: 
``` SELECT m FROM Medical m where m.type.description like :type order BY m.description ```
I don't know what is the approach in this project but omitting joins will give shorter queries. And in my opinion more readable but it's only my opinion